<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_teacher</class>
 <widget class="QWidget" name="Edit_teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="et_name_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_name_edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="et_surname_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_surname_edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="et_age_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Возраст</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_age_edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="et_faculty_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Кафедра</string>
   </property>
  </widget>
  <widget class="QComboBox" name="et_faculty_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="et_save_button">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="et_delete_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="et_add_button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="et_cancel_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
